code/game/objects/items/toys.dm

```diff
@@ -594,6 +594,23 @@
 /obj/random/plushie/item_to_spawn()
 	return pick(subtypesof(/obj/item/toy/plushie) - typesof(/obj/item/toy/plushie/fluff) - typesof(/obj/item/toy/plushie/carpplushie)) //exclude the base type.
 
+/obj/random/plushie/explosive
+	var/explosive_chance = 1 // 1% to spawn a blahbomb!
+
+/obj/random/plushie/explosive/item_to_spawn() // To override the non explosive random plushie spawn
+	return
+
+/obj/random/plushie/explosive/spawn_item()
+	var/build_path = ..()
+	var/obj/item/toy/plushie/plushie = new build_path(src.loc)
```

Comment:
build path wont be a path, it will be an already spawned object. You'll just want `plushie = ..()` here

Reply:
You're right, I confused myself by only looking at the proc above, my bad